core/esmf-aspect-meta-model-python/README.md

```diff
@@ -1,84 +1,141 @@
 
-In order to download the SAMM sources, it is required to run `poetry install` once in the `esmf-aspect-model-loader`
-module. There are two possibilities to download the SAMM files and extract the Turtle sources for the Meta Model.
+There are two possibilities to download the SAMM files and extract the Turtle sources for the Meta Model: 
+SAMM release or SAMM branch
 
-### Possibility 1 (downloading a release)
+#### Download SAMM release
 
-The `download_samm_release` script may be executed with
+This script downloads a release JAR-file from GitHub, extracts them for further usage in the Aspect Model Loader:
 
-```
+To run script, execute the next command.
+```console
 poetry run download-samm-release
-```  
+```
+The version of the SAMM release is specified in the python script.
+
+Link to all Releases: [SAMM Releases](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/releases)
 
-It downloads a release JAR-file from GitHub and extracts the SAMM Files.
-The version is specified in the python script.
+#### Download SAMM branch
 
-Link to all Releases: https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/releases
+The script uses the GitHub API and downloads the files from the `main` GitHub branch. 
 
-### Possibility 2 (downloading from the repository)
+If the script is run in a pipeline, it uses a GitHub token to authorize. If the script is run locally, 
+the API is called without a token. This may cause problems because unauthorized API calls are limited.
 
-It may happen that there is no .jar file that is up to date with the changes of the SAMM.
-This script is an alternative to the `download_samm_release.py` and extracts the files from the repository
-directly instead of using the newest release.
+Run the next command to download and start working with the Aspect Model Loader.
+```console
+poetry run download-samm-branch
+```
+Link to all branches: [SAMM Releases](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/branches)
 
-The script uses the GitHub API and downloads the files from the `main` branch. If the script is run in a
-pipeline, it uses a GitHub token to authorize. If the script is run locally, the API is called without a token.
-This may cause problems because unauthorized API calls are limited.
+### Download SAMM CLI
 
-This script can be executed with
+The SAMM CLI is a command line tool for the validation of Aspect Models and the generation of artifacts, 
+such as documentation or code, from Aspect Models.
 
+Run the next command to download the SAMM client:
+```console
+poetry run download-samm-cli
 ```
-poetry run download-samm-branch
+
+How to work with SAMM CLI from the Aspect Model Loader you can read in the chapter [SMM ClI wrapper](#samm-cli-wrapper).
+
+More detailed information about SAMM CLI functionality can be found in the 
+[SAMM CLI documentation](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html).
+
+### Download test models
```

Comment:
IMO this also does not belong in the README. Shouldn't this be done automatically when the tests are run?